src/renderer/features/wallets/ForgetWallet/model/__tests__/forget-wallet-mode.test.ts

```diff
@@ -0,0 +1,34 @@
+import { allSettled, fork } from 'effector';
+
+import { SigningType, WalletType } from '@shared/core';
+import { forgetWalletModel } from '../forget-wallet-model';
+
+jest.mock('@entities/balance', () => ({
+  useBalanceService: () => ({
+    deleteBalances: jest.fn(),
+  }),
+}));
+
+jest.mock('@app/providers', () => ({
+  useMatrix: jest.fn(),
+}));
+
+const wallet = {
+  id: 2,
+  name: 'My second wallet',
+  isActive: false,
+  type: WalletType.WATCH_ONLY,
+  signingType: SigningType.WATCH_ONLY,
+};
+
+describe('features/ForgetModel', () => {
```

Comment:
Please add tests that check:
1) is wallet actually deleted
2) is balance deleted
3) is operations deleted

Reply:
I added test for deleting wallets and account, but properly testing if balances or operations deleted requires mocking custom react hook with spy function. I tried to implement it fast but didn't succed. I'm sure it's possible, but I don't know how to do it. I think we should create separate task to research how to properly do it

Reply:
Got it. So in order to properly mock balances & operations services we need to remove existing services and use only storage service with CRUD operations which are easily mocked. Right now let's leave it as is